eservice/pdo/eservice/scripts/EServiceCLI.py

```diff
@@ -248,60 +321,58 @@ def _handle_request_(self, request) :
+            request.setHeader(b'content-length', len(response))
+            request.setHeader(b'content-type', b'application/octet-stream')
+            request.write(response)
 
         except Exception as e :
             logger.error("unknown exception packing response (Invoke); %s", str(e))
             return ErrorResponse(request, http.BAD_REQUEST, "unknown exception while packing response")
 
+        try :
+            logger.info('[%05d] finish invoke request: %s, %s, %s, %s',
+                        request.request_identifier,
+                        request.finished,
+                        request.sentLength,
+                        request.chunked,
+                        request.startedWriting)
+
+            request.finish()
+        except AttributeError as e :
+            logger.debug('[%05d] attribute error: %s', request.request_identifier, str(e))
+        except RuntimeError as e :
+            logger.debug('[%05d] runtime error: %s', request.request_identifier, str(e))
+        except :
+            logger.exception("exception during request finish")
+
+        return request
+
     ## -----------------------------------------------------------------
     def render_POST(self, request) :
         return self._defer_request_(self._handle_request_, request)
 
-## XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX
-## XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX
-class ContractEnclaveServer(Resource):
-
-    ## -----------------------------------------------------------------
-    def __init__(self, enclave, storage_url) :
-        Resource.__init__(self)
-        self.enclave = enclave
-        self.storage_url = storage_url
-
-    ## -----------------------------------------------------------------
-    def getChild(self, name ,request) :
-        logger.debug('request: %s', name)
-        if name == b'invoke' :
-            return InvokeResource(self.enclave)
-        elif name == b'verify' :
-            return VerifyResource(self.enclave)
-        elif name == b'info' :
-            return InfoResource(self.enclave, self.storage_url)
-        elif name == b'shutdown' :
-            return ShutdownResource()
-        else :
-            return NoResource()
-
 # -----------------------------------------------------------------
 # -----------------------------------------------------------------
 def StartEnclaveService(http_host, http_port, enclave, storage_url) :
     logger.info('service started on port %s', http_port)
 
-    root = ContractEnclaveServer(enclave, storage_url)
-    site = Site(root)
+    # root = ContractEnclaveServer(enclave, storage_url)
+    root = Resource()
+    root.putChild(b'invoke', InvokeResource(enclave))
+    root.putChild(b'verify', VerifyResource(enclave))
+    root.putChild(b'info', InfoResource(enclave, storage_url))
+    root.putChild(b'shutdown', ShutdownResource())
+
+    site = Site(root, logPath='/tmp/logs/logs.txt', timeout=60)
```

Comment:
the eservice fails to listen due to the logpath.
AR:
-fix logPath
-or remove it